salt/metalk8s/solutions/available.sls

```diff
@@ -0,0 +1,116 @@
+#
```

Comment:
Maybe this state should be renamed because he not really make the solutions available now :/ maybe like for archives of metalk8s `mounted.sls`

Reply:
Yeah, we'll change that later, we must first merge this asap.